sdk/recoveryservices-siterecovery/Azure.ResourceManager.RecoveryServicesSiteRecovery/api/Azure.ResourceManager.RecoveryServicesSiteRecovery.netstandard2.0.cs

```diff
@@ -5833,63 +5833,63 @@ public partial class VMwareCbtMigrationDetails : Azure.ResourceManager.RecoveryS
         public int? ResyncProgressPercentage { get { throw null; } }
         public string ResyncRequired { get { throw null; } }
         public long? ResyncRetryCount { get { throw null; } }
-        public Azure.ResourceManager.RecoveryServicesSiteRecovery.Models.ResyncState? ResyncState { get { throw null; } }
+        public Azure.ResourceManager.RecoveryServicesSiteRecovery.Models.SiteRecoveryResyncState? ResyncState { get { throw null; } }
         public System.Collections.Generic.IReadOnlyDictionary<string, string> SeedDiskTags { get { throw null; } }
         public string SnapshotRunAsAccountId { get { throw null; } }
         public string SqlServerLicenseType { get { throw null; } }
-        public string StorageAccountId { get { throw null; } }
+        public Azure.Core.ResourceIdentifier StorageAccountId { get { throw null; } }
         public System.Collections.Generic.IReadOnlyList<string> SupportedOSVersions { get { throw null; } }
-        public string TargetAvailabilitySetId { get { throw null; } }
+        public Azure.Core.ResourceIdentifier TargetAvailabilitySetId { get { throw null; } }
         public string TargetAvailabilityZone { get { throw null; } }
-        public string TargetBootDiagnosticsStorageAccountId { get { throw null; } }
+        public Azure.Core.ResourceIdentifier TargetBootDiagnosticsStorageAccountId { get { throw null; } }
         public System.Collections.Generic.IReadOnlyDictionary<string, string> TargetDiskTags { get { throw null; } }
         public string TargetGeneration { get { throw null; } }
         public string TargetLocation { get { throw null; } }
```

Comment:
Could this be AzureLocation?

Reply:
Unable to confirm if it's `AzureLocation` or not, leave it as is.